### PR TITLE
#2559. Add more augmented declarations tests. Update assertions.

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

File: LanguageFeatures/Augmentation-libraries/augmenting_declarations_A04_t05.dart [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+class A {
+  String method() => "Method";
+  String get getter => "Getter";
+  void set setter(String _) {}
+  String operator +(Object? other) => "Operator";
+}
+
+extension type ET(A id) implements A {}
+
+augment extension Ext {
```
Comment (eernstg) [inline LanguageFeatures/Augmentation-libraries/augmenting_declarations_A04_t05.dart:29]: I think this should be an extension type augmentation, not an extension augmentation.

**Review by eernstg — COMMENTED** [review]
Looks good! Just one thing: It looks like an `extension` declaration should have been an `extension type` declaration.

File: LanguageFeatures/Augmentation-libraries/augmenting_declarations_A04_t05.dart (hunk 1 above)
Comment (sgrekhov) [reply LanguageFeatures/Augmentation-libraries/augmenting_declarations_A04_t05.dart]: Of course! Thank you! Fixed

**Review by eernstg — APPROVED** [review]
LGTM.

**Pull request merged**